backend/danswer/configs/constants.py

```diff
@@ -57,6 +57,7 @@
 KV_GEN_AI_KEY_CHECK_TIME = "genai_api_key_last_check_time"
 KV_SETTINGS_KEY = "danswer_settings"
 KV_CUSTOMER_UUID_KEY = "customer_uuid"
```

Comment:
the new value is stored in postgres as a kv so it stays consistent once generated